File: python/prism_sdk/local_instrument_execution_integrity_inference.py
"""Lab P32 local inference instrument-execution integrity feature F01."""
from .instrument_execution_integrity_support import InstrumentExecutionRequest4,InstrumentExecutionCard7,InstrumentExecutionIntegrityError,manifest,qualify
FEATURE_ID="AFA-lab-P32-F01";CONTRACT_VERSION="lab-local_instrument_execution_integrity_inference/1.0"
def local_instrument_execution_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="inference")
def qualify_local_instrument_execution_integrity_inference(request:InstrumentExecutionRequest4)->InstrumentExecutionCard7:return qualify(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="local",mode="inference")

File: crates/lab/src/local_instrument_execution_integrity_inference.rs
//! Lab P32 local inference instrument-execution integrity feature.
use super::instrument_execution_integrity_support::{manifest,qualify,InstrumentExecutionCard7,InstrumentExecutionIntegrityError,InstrumentExecutionRequest4};
pub const FEATURE_ID:&str="AFA-lab-P32-F01";pub const CONTRACT_VERSION:&str="lab-local_instrument_execution_integrity_inference/1.0";
pub fn local_instrument_execution_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"local","inference")}
pub fn qualify_local_instrument_execution_integrity_inference(request:&InstrumentExecutionRequest4)->Result<InstrumentExecutionCard7,InstrumentExecutionIntegrityError>{qualify(request,FEATURE_ID,CONTRACT_VERSION,"local","inference")}
